odata-csdl/12 Action and Function.md

```diff
@@ -285,8 +285,8 @@ of the collection and specifies whether the collection MAY contain
 `null` values.
 
 For single-valued return types the value `true` means that the action or
-function MAY return a single `null` value. The value `false` means that
-the action or function will never return a `null` value and instead will
+function overload MAY return a single `null` value. The value `false` means that
+the action or function overload will never return a `null` value and instead will
```

Comment:
Make analogous change for the XML variant.
Must all function overloads with the same name have the same return type nullability?

Reply:
I assumed that the full return type (including nullability and type facets) must be identical. Added this above.